django/templatetags/static.py

```diff
@@ -150,5 +156,14 @@ def do_static(parser, token):
     return StaticNode.handle_token(parser, token)
 
 
+@register.simple_tag
```

Comment:
This shouldn't be here; `static()` is a helper function (admittedly, I think, undocumented and I suspect there only for the benefit of admin's helper `admin_static` template library, although it may need a deprecation cycle to remove) and isn't a tag. You removed the static tag (the old `do_static()` function, decorated), which is needed.